source/src/plugins/MeanWithinExpectedTest.cc

```diff
@@ -87,43 +94,106 @@ namespace dqm4hep {
     //-------------------------------------------------------------------------------------------------
 
     StatusCode MeanWithinExpectedTest::readSettings(const TiXmlHandle xmlHandle) {
-      RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=, XmlHelper::readParameter(xmlHandle, "ExpectedMean", m_expectedMean));
-
-      RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=,
-                       XmlHelper::readParameter(xmlHandle, "MeanDeviationLower", m_meanDeviationLower,
```

Comment:
Rename the parameter to DeviationLower as we allow for testing other properties. Same for DeviationUpper